engine/src/hopeit/app/api.py

```diff
@@ -151,20 +172,25 @@ def _event_api(
     return method_spec
 
 
-def event_api(title: Optional[str] = None,
+def event_api(summary: Optional[str] = None,
+              description: Optional[str] = None,
```

Comment:
This param if its new should be added to the end (before title) to prevent breaking existing positional usages. Ivwould make it named argument (after *).